Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -216,7 +216,8 @@ public abstract partial class BaseMeshSync : MonoBehaviour, IObservable<MeshSync
 
         #region Properties
         
-        private int  currentSessionId                 = -1;
+        [SerializeField] private int  currentSessionId = -1;
```

Comment:
We could also make this persist editor restarts. Here is an example https://github.com/unity3d-jp/MeshSync/blob/dev/Editor/Scripts/Server/EditorServerSettings.cs

Reply:
As we discussed I'll add this to the technical backlog to think about the best place to persist this setting